transformations/visual_attack_letters/transformation.py

```diff
@@ -0,0 +1,107 @@
+  'x': ['ⅹ', 'х', 'ẋ', 'ӿ', 'ẍ', 'ҳ', 'ӽ'],
+  'y': ['у', 'ỵ', 'ү', 'ẏ', 'ȳ', 'ӯ', 'ÿ', 'ӱ', 'ỹ', 'ƴ', 'ў', 'ý', 'ŷ', 'ỷ', 'ỳ', 'ӳ', 'ẙ'],
+  'z': ['ᴢ', 'ẓ', 'ẕ', 'ż', 'ƶ', 'ȥ', 'ʐ', 'ź', 'ẑ', 'ʑ', 'ƨ', 'ž'],
+  
+  'A': ['Α', 'А', 'Ạ', 'ᾼ', 'Ḁ', 'Ἀ', 'Ἁ', 'ᾈ', 'ᾉ', 'Ȧ', 'Ά', 'Ά', 'Ⱥ', 'Ą', 'Ả', 'ᕕ', 'À', 'Ẵ', 'Ằ', 'Ắ', 'Ᾰ', 'Á', 'Ȃ', 'Ã', 'Ӓ', 'Ä', 'Ẳ', '₳', 'Ȁ', 'Ẫ', 'Â', 'ᕖ', 'Ậ', 'Ᾱ', 'Ā', 'Ǎ', 'Å', 'Å', '4', 'Ӑ', 'Ă', 'Ặ', 'Ầ', 'Ấ', 'Ǡ', 'Ǟ', 'Ẩ', 'Ὰ'],
+  'B': ['Β', 'В', 'Ḅ', 'Ḇ', 'Ḃ', '8', 'Ƀ', 'ß', 'β', '฿'],
+  'C': ['Ⅽ', 'Ϲ', 'С', 'Ҫ', 'Ç', 'Ҁ', 'ᑕ', 'Ċ', 'ᕦ', 'ᑢ', 'Ϛ', 'ⵛ', 'ⵎ', 'Ȼ', 'ᑖ', 'Ć', 'ᕧ', 'ᑤ', '₵', 'ᕩ'],
+  'D': ['Ⅾ', 'ᗞ', 'Ḏ', 'Ḍ', 'Ḓ', 'Ḑ', 'ↁ', 'Ḋ', 'Ð', 'Ɖ', 'Đ', 'Ď', 'Ɒ'],
+  'E': ['Ε', 'ⴹ', 'Е', 'Ẹ', 'ⵟ', 'Ȩ', 'Ḛ', 'Ę', 'Ḙ', 'Ė', 'Ɛ', 'Ԑ', 'ℇ', 'Є', 'Ẻ', 'È', 'Ӗ', 'Ĕ', 'Ѐ', 'É', 'Ȇ', 'Ƹ', 'Ḝ', 'Ẽ', '⋿', 'Ё', 'Ë', '∃', 'ⴺ', 'Ǝ', '£', 'Ȅ', 'Ễ', 'Ê', 'Ệ', 'Σ', 'ⵉ'],
+  'F': ['Ϝ', 'Ḟ', 'ߓ', 'Ғ'],
+  'G': ['Ģ', 'Ǥ', 'Ԍ', 'Ġ', 'Ğ', 'Ǵ', 'Ḡ', 'Ĝ'],
+  'H': ['Η', 'Н', 'ᕼ', 'Ḥ', 'Ḫ', 'ῌ', 'Ḣ', 'Ḩ', 'Ӈ', 'Ң', 'Ⱨ', 'Ḧ', 'Ĥ', 'Ȟ', 'Ӊ', 'ℍ'],
+  'I': ['Ι', 'Ӏ', 'ⵏ', 'І', 'Ⅰ', 'Ị', 'ߊ', 'ⵑ', 'ا', 'ǃ', 'Į', 'ⅼ', 'ӏ', 'إ','׀', 'Ḭ'],
+  'J': ['Ј', 'Ɉ', 'յ', 'Ĵ', 'ȷ'],
+  'K': ['Κ', 'K', 'Ķ', 'Ḳ', 'Ḵ', '₭', 'Ƙ', 'Ҝ', 'Ḱ', 'К', 'Ԟ', 'Ҟ', 'Ϗ', 'қ', 'Ǩ', 'ⱪ', 'ҟ', 'Ⱪ', 'ԟ', 'ҝ', 'κ', 'к', 'Ќ', 'ⴿ'],
+  'L': ['ᒪ', 'Ⅼ', 'Ļ', 'Լ', 'Ḷ', 'Ḻ', 'Ŀ', 'ᒷ', 'Ľ', 'Ḽ', 'ʟ', 'ւ'],
+  'M': ['Ⅿ', 'Μ', 'М', 'Ṃ', 'Ṁ', 'Ϻ', 'Ḿ', 'Ɱ', 'Ӎ', 'ᴍ', 'м'],
+  'N': ['Ν', 'Ņ', 'Ṉ', 'Ṇ', 'Ṋ', 'Ṅ', 'Ǹ', 'Ń', 'Ñ', 'Ň', 'Ɲ', 'Ŋ'],
+  'O': ['Ο', 'Օ', 'О', 'Ọ', 'Ǫ', 'Ơ', 'Θ', 'Ϙ', 'Ợ', 'Ɵ', 'ϴ', 'Ө', 'Ѳ', 'Ȯ', 'Q', 'Ԛ', 'Ό', 'Ό', 'Ὀ', 'Ò', 'Ŏ', '⊙', 'Ờ', 'Ȏ', 'Ó', 'Ỏ', 'Ȍ', '⊝', 'Õ', 'Ṍ', 'Ṏ', 'Ӧ', 'Ö', '⊘', '⊖', 'Ớ', 'Ở', '⊜', '⊛', 'Ỡ', '⊚', 'ʘ', 'Ǿ', 'Ô', 'Ỗ', 'Ộ', 'Ӫ', '⊗', '⊕', 'Փ', 'Ṑ', 'Ō', 'Ṓ', 'Ǒ', 'Ǭ', 'Ὁ', 'Ø', 'Ѻ'],
+  'P': ['Ρ', 'Р', 'Ҏ', 'ᑭ', 'ᕈ', 'Ṗ', 'ᑷ', '₱', 'Ᵽ', 'ᑮ', 'ᑹ', 'Ṕ', 'ᕉ', 'ᒆ'],
+  'Q': ['Ԛ', 'ℚ'],
+  'R': ['Ŗ', 'Ṛ', 'Ṟ', 'Ɽ', 'Ɍ', 'Ṙ', 'Ŕ', 'Ȓ', 'Ṝ', 'Ȑ', '℞', 'Ř', 'Я'],
+  'S': ['Ѕ', 'Ș', 'Ṣ', 'Ş', 'Ṡ', 'Ṩ', '5', 'Ś', 'Ƽ', 'Ŝ', 'Ṥ', 'Ṧ', 'Š', 'Ƨ'],
+  'T': ['Т', 'Τ', 'Ț', 'Ṭ', 'Ṯ', 'Ṱ', 'Ţ', 'Ŧ', 'Ʈ', 'Ṫ', 'Ҭ', '₮', 'Ԏ', 'Ƭ', '₸', 'Ť', 'ⴶ', 'ͳ', 'ߠ', '☨', '⊺', '☦'],
+  'U': ['Ս', 'ᑌ', 'Ṳ', 'Ụ', 'Ų', 'Ṵ', 'ᑘ', 'Ṷ', 'ⵡ', 'Ư', 'Ự', 'Џ', 'Ա', 'Ù', 'Ʉ', 'ᕟ', 'ᕞ', 'Ŭ', 'Ȗ', 'Ú', 'Ủ', 'Ȕ', 'Ů', 'Ṹ', 'Ũ', 'ᑧ', 'Ü', 'Ǘ', 'Ǜ', 'Ǚ', 'Ừ', 'Ц', 'Ứ', 'Ử', 'Û', '∪', 'Ū', 'Ǔ', 'Մ', '⊎', '⊍', '⊌'],
+  'V': ['Ⅴ', 'ᐯ', 'ⴸ', 'Ṿ', 'ᐻ', 'Ѵ', '∨', '⊽', 'Ѷ', '⋎', '℣', 'Ṽ'],
+  'W': ['Ԝ', 'Ẉ', 'Ẇ', 'Ẁ', 'Ⱳ', '₩', 'Ẅ', 'Ẃ', 'Ŵ', 'Ɯ'],
+  'X': ['Х', 'Χ', 'ⵝ', 'Ⅹ', 'Ӿ', 'ⵅ', 'ⴴ', 'Ẋ', 'ⴳ', 'Ẍ', 'Ӽ', 'Ҳ', '✘', '☓', 'ⴵ'],
+  'Y': ['Ү', 'Υ', 'Ỵ', 'Ẏ', 'Ƴ', 'Ỷ', 'Ý', 'Ῠ', 'Ỳ', 'Ɏ', 'Ỹ', 'Ÿ', 'Ϋ', 'Ŷ', 'Ȳ', 'Ῡ', 'ϒ'],
+  'Z': ['Ζ', 'Ẓ', 'Ẕ', 'Ƶ', 'Ż', 'Ȥ', 'Ź', 'ƻ', 'Ẑ', 'Ž']
+}
+
+
+class VisualAttackLetters(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION, TaskType.TEXT_TAGGING]
+    languages = ["en"]
+
```

Comment:
Add a keywords attribute following the [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) section recommendations.